index.html

```diff
@@ -76,7 +82,10 @@ <h1>
     <section class="works">
       <div class="card-works">
         <div class="snapshoot-portfolio">
-          <img src="./images/Snapshoot Portfolio.png" alt="website" />
```

Comment:
The design looks pretty well, but you can improve it by fixing the font weight, color, size, and padding for the card.

![image](https://user-images.githubusercontent.com/34779928/194422370-3202b57a-3fad-406d-8730-507e887c37ac.png)